include/fast_float/fast_float.h

```diff
@@ -3,13 +3,15 @@
 
 
 #include <system_error>
-#include <version>
 
+#if !defined(CXX20_CONSTEXPR) && defined(__has_include) && __has_include(<version>)
```

Comment:
This is not portable code, please see rationale at https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005finclude.html